packages/spark-core/base/inputs/_text-input.scss

```diff
@@ -0,0 +1,97 @@
+// ==================================================================
+// Input Component
+// ==================================================================
+.sprk-b-TextInput {
+  font-size: $font-size-body-two;
+  font-weight: $font-weight-body-two;
+  line-height: $line-height-body-two;
+  outline: 0;
+  padding: 3px $space-s;
+  box-shadow: $input-box-shadow;
+}
+
+.sprk-b-TextInput[disabled] {
+  background-color: $gray-light;
```

Comment:
Scales has  $input-disabled-background-color and $input-disabled-text-color for us to use.